HW02_B_Mimic_starter.Rmd

```diff
@@ -41,6 +41,7 @@ data("diamonds")
 
 Using the diamonds dataset, make this graph:
 
+
 ```{r graph1 code, echo=FALSE}
 
 ggplot(diamonds, aes(cut)) + 
```

Comment:
Very solid plot mimic! The only ways it doesn't seem to match up is that the line break falls after, not before, "by" in the subtitle. And you seem to be missing the rectangle highlighting the the Ideal diamonds. I made that using the annotate() function with the "rect" argument: annotate("rect", xmin = 4.5, xmax = 5.5, ymin = 0, ymax = 5000, alpha = 0.4), if that helps!